Makefile

```diff
@@ -62,7 +62,7 @@ install-githook:
 	./tools/git-hooks/setup.sh
 
 lint-autocorrect:
-	(swiftlint autocorrect --format)
+	(swiftlint --fix --format)
```

Comment:
Will need to fix the git hook script as well as it uses `autocorrect`. 
In addition, I think it would good to apply git hook scripts by changing the local git config to point to a `.githooks` folder we have in source control. It's something we did for the Edge Identity and Edge Consent projects.
https://github.com/adobe/aepsdk-edgeidentity-android/blob/dev/Makefile#L15
This allows updates to the git hook scripts without developers needing to recopy the pre-commit script to their local .git/hooks directory.